docs: add docs for CST commands (#8090)
confluentinc/ksql: docs/developer-guide/ksqldb-reference/create-stream.md

If the IF NOT EXISTS clause is present, the statement won't fail if a 
stream with the same name already exists.

### Source streams

> A stream can be created as read-only if the SOURCE clause is provided. INSERTS statements and the

Reviewer: ```suggestion
A stream can be created as read-only if the SOURCE clause is provided. For a read-only stream, INSERT statements, and the

```

> DELETE TOPIC on DROP STREAM statements won't be permitted. Source streams do not support pull

Reviewer: ```suggestion
DELETE TOPIC on DROP STREAM statements aren't permitted. Source streams don't support pull

```

> queries currently. Only source tables support running pull queries on them.

Reviewer: ```suggestion
queries. Only source tables support running pull queries on them.

```

> queries currently. Only source tables support running pull queries on them.

Reviewer 2: Pull queries don't work on any stream, so I'm not sure if we need to specify this here.

> To disable the SOURCE stream feature, you can set `ksql.source.table.materialization.enabled` to

Reviewer: ```suggestion
To disable the SOURCE stream feature, set `ksql.source.table.materialization.enabled` to

```
